From c4be9bb5d281a7dad9fc409cde43cab349a369b0 Mon Sep 17 00:00:00 2001
From: Manuel Bachmann <manuel.bachmann@open.eurogiciel.org>
Date: Thu, 6 Mar 2014 12:22:25 +0100
Subject: [PATCH 06/13] packaging: add weston-editor & weston-subsurfaces
 clients.

The editor client has virtual keyboard API, and subsurfaces
the new subsurfaces API.

Change-Id: I41bb14bb673a1c3bf9d6b063ec038885e9d94621
Signed-off-by: Manuel Bachmann <manuel.bachmann@open.eurogiciel.org>
---
 packaging/weston.spec | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/packaging/weston.spec b/packaging/weston.spec
index 84d2cb8..aeb57af 100644
--- a/packaging/weston.spec
+++ b/packaging/weston.spec
@@ -40,6 +40,7 @@ BuildRequires:  pkgconfig(libffi)
 BuildRequires:  pkgconfig(libsystemd-login)
 BuildRequires:  pkgconfig(libudev) >= 136
 BuildRequires:  pkgconfig(mtdev) >= 1.1.0
+BuildRequires:  pkgconfig(pangocairo)
 BuildRequires:  pkgconfig(pixman-1)
 BuildRequires:  pkgconfig(poppler-glib)
 BuildRequires:  pkgconfig(wayland-client)
@@ -101,10 +102,12 @@ install -m 755 clients/weston-flower %{buildroot}%{_bindir}
 install -m 755 clients/weston-image %{buildroot}%{_bindir}
 install -m 755 clients/weston-cliptest %{buildroot}%{_bindir}
 install -m 755 clients/weston-dnd %{buildroot}%{_bindir}
+install -m 755 clients/weston-editor %{buildroot}%{_bindir}
 install -m 755 clients/weston-smoke %{buildroot}%{_bindir}
 install -m 755 clients/weston-resizor %{buildroot}%{_bindir}
 install -m 755 clients/weston-eventdemo %{buildroot}%{_bindir}
 install -m 755 clients/weston-clickdot %{buildroot}%{_bindir}
+install -m 755 clients/weston-subsurfaces %{buildroot}%{_bindir}
 install -m 755 clients/weston-transformed %{buildroot}%{_bindir}
 install -m 755 clients/weston-fullscreen %{buildroot}%{_bindir}
 install -m 755 clients/weston-calibrator %{buildroot}%{_bindir}
@@ -153,10 +156,12 @@ getent group weston-launch >/dev/null || %{_sbindir}/groupadd -o -r weston-launc
 %_bindir/weston-image
 %_bindir/weston-cliptest
 %_bindir/weston-dnd
+%_bindir/weston-editor
 %_bindir/weston-smoke
 %_bindir/weston-resizor
 %_bindir/weston-eventdemo
 %_bindir/weston-clickdot
+%_bindir/weston-subsurfaces
 %_bindir/weston-transformed
 %_bindir/weston-fullscreen
 %_bindir/weston-calibrator
-- 
1.8.1.4

